fedimg/consumers.py

```diff
@@ -99,19 +98,19 @@ def consume(self, msg):
                 compose_metadata, 'images', 'payload', 'images', 'CloudImages',
                 'x86_64')
         else:
-            images_meta = get_value_from_dict(
+            cloud_meta = get_value_from_dict(
                 compose_metadata, 'images', 'payload', 'images',
                 'Cloud', 'x86_64')
-            if images_meta is None:
-                images_meta = get_value_from_dict(
-                    compose_metadata, 'images', 'payload',
-                    'images', 'AtomicHost', 'x86_64')
+            atomic_meta = get_value_from_dict(
+                compose_metadata, 'images', 'payload',
+                'images', 'AtomicHost', 'x86_64')
+
+            if (cloud_meta and atomic_meta) is None:
+                images_meta = cloud_meta or atomic_meta
             else:
-                images_meta.extend(get_value_from_dict(
-                    compose_metadata, 'images', 'payload',
-                    'images', 'AtomicHost', 'x86_64'))
+                images_meta = cloud_meta + atomic_meta
 
-        if images_meta is None:
```

Comment:
The changes here are mostly cosmetic right? There is no bug getting fixed here?

Reply:
Yes, these changes are cosmetic changes and fixes the broken tests.

Reply:
Since the changes don't actually change any functionality, how do they fix broken tests?